README.md

```diff
@@ -203,6 +214,76 @@ you are recommended to take a look at documentation for the `holochain_wind_tunn
 your scenarios and example of how to use them. This will help you get started much more quickly than starting from scratch. There is
 also a tips section below which you might find helpful as you run into questions.
 
+
+### Writing Scenarios for Kitsune
+
+> [!NOTE]
+> Writing scenarios requires some knowledge of `wind-tunnel`'s methodology as well as an overview of how Kitsune works. That is assumed knowledge for this section!
+
+Writing a Kitsune Wind Tunnel scenario is relatively straight forward. The Kitsune client defines three common functions for the developer. A chatter can be `create`d, it can `join_chatter_network` and it can `say` a list of messages. As long as a chatter has not joined the network, it won't receive messages from other peers and will also not send messages to them. Once joined, it starts receiving and sending messages it has said.
```

Comment:
We are relying on gossip right? So it won't receive data from peers until it joins but it will get that data once it does join?

Reply:
Yes, I'm assuming just gossip here.

Reply:
I've mentioned that now.

Reply:
Sorry, already squashed: https://github.com/holochain/wind-tunnel/pull/153/commits/eb28bc8b87c8d38f8e426f15c15d00a9f4336b94